pallets/ethereum-transactions/src/lib.rs

```diff
@@ -627,7 +627,6 @@ pub trait CandidateTransactionSubmitter<AccountId> {
     ) -> DispatchResult;
 
     /// Sets a transaction Id. This is only enabled for benchmarks
-    #[cfg(feature = "runtime-benchmarks")]
```

Comment:
Why was this removed?

Reply:
because otherwise the compiler outputs an error `set_transaction_id not found on trait...`